api-tests/features/inherited-identity/extended-scenarios.feature

```diff
@@ -0,0 +1,290 @@
+
+    # Return journey with inherited identity for the same user
+    When I start a new 'medium-confidence-pcl200-pcl250' journey with inherited identity '<identity-details>'
+    Then I get an OAuth response
+    And an 'IPV_INHERITED_IDENTITY_VC_RECEIVED' audit event was recorded [local only]
+    When I use the OAuth response to get my identity
+    Then I get a '<expected-identity>' identity
+
+    Examples:
+    | inherited-identity | identity-details             | expected-identity |
+    | PCL200             | alice-vot-pcl200-no-evidence | PCL200            |
+    | PCL250             | kenneth-vot-pcl250-passport  | PCL250            |
+
+  Scenario Outline: Successfully migrates <inherited-identity> HMRC profile when user has failed F2F (no CI)
+    Given I start a new 'medium-confidence' journey
+    Then I get a 'page-ipv-identity-document-start' page response
+    When I submit an 'end' event
+    Then I get a 'page-ipv-identity-postoffice-start' page response
+    When I submit a 'next' event
+    Then I get a 'claimedIdentity' CRI response
+    When I submit 'kenneth-current' details to the CRI stub
+    Then I get an 'address' CRI response
+    When I submit 'kenneth-current' details to the CRI stub
+    Then I get a 'fraud' CRI response
+    When I submit 'kenneth-score-2' details to the CRI stub
+    Then I get a 'f2f' CRI response
+    When I get an error from the async CRI stub
+    Then I get a 'page-face-to-face-handoff' page response
+
+    # Return journey
+    When I start a new 'medium-confidence' journey and return to a 'pyi-f2f-technical' page response
+    When I submit a 'end' event
+    Then I get an OAuth response
+    When I use the OAuth response to get my identity
+    Then I get a 'P0' identity without a TICF VC
+
+    # New journey with inherited identity for the same user
+    Given I start a new 'medium-confidence-pcl200-pcl250' journey with inherited identity '<identity-details>'
+    Then I get an OAuth response
+    When I use the OAuth response to get my identity
+    Then I get a '<expected-identity>' identity
+
+    Examples:
+      | inherited-identity | identity-details             | expected-identity |
+      | PCL200             | alice-vot-pcl200-no-evidence | PCL200            |
+      | PCL250             | kenneth-vot-pcl250-passport  | PCL250            |
+
+  Scenario Outline: Successfully completes reprove identity journey with a <inherited-identity> HMRC profile and receives a P2
+    Given I start a new 'medium-confidence' journey
+    Then I get a 'page-ipv-identity-document-start' page response
+    When I submit an 'appTriage' event
+    Then I get a 'dcmaw' CRI response
+    When I submit 'kenneth-passport-valid' details to the CRI stub
+    Then I get a 'page-dcmaw-success' page response
+    When I submit a 'next' event
+    Then I get an 'address' CRI response
+    When I submit 'kenneth-current' details to the CRI stub
+    Then I get a 'fraud' CRI response
```

Comment:
I'm not sure this 'partial' journey is adding anything useful?

Reply:
This is like if the user has partial vc's already in ipv core and then user try to inherit ID scenario and think of merging this scenario already in some other case then we wont need it again